presto-main/src/main/java/com/facebook/presto/execution/ClusterSizeMonitor.java

```diff
@@ -57,7 +59,13 @@
     private int currentCount;
```

Comment:
should we rename this to `currentWorkerCount` to make it clearer?

Reply:
Sure.